create-ladder.js

```diff
@@ -26,4 +26,40 @@ const addHandlersToElements = () => {
     })(i);
   }
 }
+
+const addStep = document.getElementById('addStep');
```

Comment:
Again, I know its not on this pull, but in earlier I asked you to find a better name than `addHandlersToElements` [above](https://github.com/CYPIAPT-LNDSE/anxiety-ladder/pull/45/files#diff-cbc266616471666b17494f515a7f47a8R19). Sorry I missed that you had not changed this when I merged.